cmd/prometheus/main.go

```diff
@@ -1005,3 +1016,45 @@ func (opts tsdbOptions) ToTSDBOptions() tsdb.Options {
 		MaxBlockDuration:       int64(time.Duration(opts.MaxBlockDuration) / time.Millisecond),
 	}
 }
+
+func initTracing(logger log.Logger) (io.Closer, error) {
+	// Set tracing configuration defaults.
+	cfg := &jaegercfg.Configuration{
+		ServiceName: "prometheus",
+		Disabled:    true,
+	}
+
+	// Override defaults from environment variables. Available options can be seen here:
```

Comment:
Override isn't the right word, I'd remove that first sentance.

Reply:
Removed